test/rubygems/test_gem_commands_open_command.rb

```diff
@@ -21,7 +21,7 @@ def gem(name, version = "1.0")
 
   def test_execute
     @cmd.options[:args] = %w[foo]
-    @cmd.options[:editor] = "#{Gem.ruby} -eexit --"
```

Comment:
Do not overwrite this https://github.com/ruby/ruby/commit/ceea1ea8af2376845f5f6164e9432e5b9845c033

Reply:
I have already ported ceea1ea at https://github.com/rubygems/rubygems/commit/d35448a98c86e198860a4b53c16873941d07b790, so I can confirm I'm not planning to revert it :)